samples/kvsWebRTCClientMaster.c

```diff
@@ -171,13 +169,23 @@ PVOID sendVideoPackets(PVOID args)
         CHK_STATUS(readFrameFromDisk(frame.frameData, &frameSize, filePath));
 
         frame.presentationTs += SAMPLE_VIDEO_FRAME_DURATION;
-        CHK_STATUS(writeFrame(pSampleConfiguration->pVideoRtcRtpTransceiver, &frame));
+
+        if (!ATOMIC_LOAD_BOOL(&pSampleConfiguration->updatingSampleStreamingSessionList)) {
+            ATOMIC_INCREMENT(&pSampleConfiguration->streamingSessionListReadingThreadCount);
+            for (i = 0; i < pSampleConfiguration->streamingSessionCount; ++i) {
+                status = writeFrame(pSampleConfiguration->sampleStreamingSessionList[i]->pVideoRtcRtpTransceiver, &frame);
```

Comment:
pSampleConfiguration->sampleStreamingSessionList[i] might be NULL right? If the viewer session is terminated?